steps/firebase-app-distribution/0.2.0/step.yml

```diff
@@ -0,0 +1,92 @@
+title: '[BETA] Firebase App Distribution'
+summary: |
+  [BETA] Distributes your builds via Firebase App Distribution
+description: |
+  [BETA] Uploads your app binaries to Firebase and sends email to your testers.
+website: https://github.com/guness/bitrise-step-firebase-app-distribution
+source_code_url: https://github.com/guness/bitrise-step-firebase-app-distribution
+support_url: https://github.com/guness/bitrise-step-firebase-app-distribution
```

Comment:
for the next version pls update to: `https://github.com/guness/bitrise-step-firebase-app-distribution/issues`